holoviews/operation/timeseries.py

```diff
@@ -104,9 +113,9 @@ def _process_layer(self, element, key=None):
         ys = element.dimension_values(1)
 
         # Calculate the variation in the distribution of the residual
-        avg = pd.Series(ys).rolling(window, center=True).mean()
+        avg = pd.Series(ys).rolling(**self._roll_kwargs()).mean()
         residual = ys - avg
-        std = pd.Series(residual).rolling(window, center=True).std()
+        std = pd.Series(residual).rolling(**self._roll_kwargs()).std()
```

Comment:
Good point; apparently .std() is only defined for a rectangular window, so I've moved the window type specification into `rolling` itself.